### PR TITLE
✨Added gamemodes, location, supported languages and removed some useless bits. ✨[LendMark Network (formally LendMark)]

### PR DESCRIPTION
**Pull request by SlenkyDev**
## Type of change

- [ ] Add new directory for a server
- [x] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/lendmark/manifest.json [hunk 1]
```diff
@@ -1,15 +1,18 @@
...
+    ],
+    "supported_languages": [
+        "sk",
+        "cz"
     ],
     "social": {
       "web": "https://lendmark.sk/",
       "web_shop": "https://store.lendmark.sk/",
-      "discord": "https://discord.com/invite/kaMfqnvaqC",
+      "discord": "https://discord.lendmark.sk",
```
Comment (scrummer) [inline minecraft_servers/lendmark/manifest.json:15]: Please change this back to an official Discord invite link (referring to [docs/Manifest.md#discord-url](https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#discord-url)

File: minecraft_servers/lendmark/manifest.json [hunk 2]
```diff
@@ -18,5 +21,25 @@
...
     }
+  },
+  "location": {
+    "city": "Bratislava",
+    "country": "Slovakia",
+    "country_code": "SK"
+  },
+  "discord": {
+    "server_id": 865900839481901066,
+    "rename_to_minecraft_name": true
```
Comment (scrummer) [inline minecraft_servers/lendmark/manifest.json:43]: Please change this value to `false` (currently no support for non-partner servers)

**Review by scrummer — CHANGES_REQUESTED** [review]
Please in general:
- Have a look at the formatting (it does not matter if the indentation is 2 or 4 spaces, but keep it consistent)

**Review by scrummer — CHANGES_REQUESTED** [review]
Indentation looks good now but you removed all the whitespaces between keys and values. You can keep those :)

**Comment by SlenkyDev** [comment]
> Indentation looks good now but you removed all the whitespaces between keys and values. You can keep those :)

I used a JSON formatter for that because I suck at formatting. Also removed the gamemodes as we change them quite often and currently we don't have many.

**Comment by scrummer** [comment]
@SlenkyDev Then I would recommend you this one, looks like it does everything right (check that you've selected 4 or 2 space indentation!)

**Comment by SlenkyDev** [comment]
Thanks!

**Comment by scrummer** [comment]
Looks good now, thank you very much!

**Pull request merged**